src/lib/datepicker/datepicker.ts

```diff
@@ -22,16 +23,19 @@ import {MdDialogRef} from '../dialog/dialog-ref';
 import {PositionStrategy} from '../core/overlay/position/position-strategy';
 import {
   OriginConnectionPosition,
-  OverlayConnectionPosition
-} from '../core/overlay/position/connected-position';
+  OverlayConnectionPosition,
+  RepositionScrollStrategy,
+  BlockScrollStrategy,
```

Comment:
The demo page actually ends up blocking scroll on whenever a connected overlay is open, because the scrollable container is the `md-sidenav-container`.